modules/view-kubernetes-secrets.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="view-kubernetes-secrets_{context}"]
+= Viewing Kubernetes secrets
+
+[role="_abstract"]
+View Kubernetes secrets in use in your environment and identify deployments using those secrets.
+
+.Procedure
+. Navigate to the RHACS portal and click *Configuration Management* from the left-hand navigation menu.
+. On the *Secrets Most Used Across Deployments* widget, select *View All*.
+Alternatively, select *Application & Infrastructure* -> *Secrets* from the *Configuration Management* view header.
+The *Secrets* view displays a list of Kubernetes secrets.
+. Select a row to view more details.
+The information appears in a panel on the right.
```

Comment:
You could potentially remove this line because this would be self-explanatory in the UI and it might also change position over time.